toolkit/docs/building/prerequisites-mariner.md

```diff
@@ -1,18 +1,18 @@
 
 # Build Requirements on `Mariner`
 
-## Requirements were validated on `Mariner 1.0`
+## Requirements were validated on `Mariner 2.0`
 
-This works regardless of installing `core` or `full` version of mariner.
-If you installed `full` version of mariner, some of these packages are already installed, so less packages will be downloaded.
-
-Requirements for building images with a toolkit:
+This page lists host machine requirements for building with the CBL-Mariner toolkit. They cover building the toolchain, packages, and images on a Mariner host.
 
 ```bash
 # Install required dependencies.
-sudo dnf -y install git make tar rpm-build gcc glibc-devel binutils \
```

Comment:
My only concern is the removal of `gcc`. I'd just want to make sure, that if it's absent from the system, the toolchain still builds fine.

Reply:
gcc is present in the Mariner 2.0 marketplace image by default. I was able to remove it, but it's re-installed as a dependency of the other packages. We should not need it on the host to build the toolchain - the initial toolchain bootstrap uses a Mariner container image with gcc. And the Ubuntu marketplace image builds fine without gcc installed.